### PR TITLE
Fix count arg in ACI deployments

### PR DESCRIPTION
**Pull request by DomAyre**
For deploying multiple ACIs, we actually want to deploy multiple container groups each with a single container.

Spun out of #4825

### CODE REVIEW

File: scripts/azure_deployment/arm_aci.py [hunk 1]
```diff
@@ -205,77 +205,79 @@ def make_aci_deployment(parser: ArgumentParser) -> Deployment:
...
+            deployment_name = args.deployment_name
+            container_name = args.deployment_name
+            container_image = args.aci_image
+            command = make_dev_container_command(args)
+            with_volume = args.aci_file_share_name is not None
+            containers = [
+                make_dev_container(
+                    i, container_name, container_image, command, args.ports, with_volume
+                )
+                for i in range(args.count)
```
Comment (jumaffre) [inline scripts/azure_deployment/arm_aci.py:220]: This should go then?

Comment (DomAyre) [reply scripts/azure_deployment/arm_aci.py:220]: Whoops yeah that's a merge conflict typo, good spot

Comment (DomAyre) [reply scripts/azure_deployment/arm_aci.py]: https://github.com/microsoft/CCF/pull/4854/commits/ff816e90f624ca165b297fe1ec3312ea9057e3db

**Review by takuro-sato — APPROVED** [review]

**Comment by takuro-sato** [comment]
I think we don't have to care about 'Attestation COntainer' failure since it's related to https://statics.teams.cdn.office.net/evergreen-assets/safelinks/1/atp-safelinks.html

**Pull request merged**